lisa/operating_system.py

```diff
@@ -569,7 +569,7 @@ def _initialize_package_installation(self) -> None:
         #  Basic_A1, Standard_A5, Standard_A1_v2, Standard_D1
         # redhat rhel 7-lvm 7.7.2019102813 Basic_A1 cost 2371.568 seconds
         # redhat rhel 8.1 8.1.2020020415 Basic_A0 cost 2409.116 seconds
-        cmd_result = self._node.execute("yum -y update", sudo=True, timeout=3600)
+        cmd_result = self._node.execute("yum update", sudo=True, timeout=3600)
```

Comment:
Move it to a separated commit, and add commit description to explain the change.